### DISCUSSION TITLE
How to deploy to Vercel

### DISCUSSION THREAD

**Discussion by boston2029** [UseInterstellar/Interstellar#802] [Q&A]
I forked the repo and tried deploying but the function invocation failed, can Vercel support be added? I see people deploy it to vercel every day but i guess this doesn't have a vercel.json? or there's no button or what. please help, thanks!

**Comment by 0mhx** [comment]
vercel support is impossible now. They broke websockets, which would break a lot of websites, and their node compat has been scaled back.
